app/serializers/chatroom_messages_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class ChatroomMessagesSerializer < ActiveModel::Serializer
+  attributes :id, :body, :sent_by, :created_at
+
+  def sent_by
+    {
+      user: self.object.sender == current_user ? 'self' : 'other'
```

Comment:
Here you could also implement a method sender(user) (or similarly named) on the chatroom model